File: coding/fizzbuzz/cpp/fizzbuzz.cpp
#include <iostream>

void fizzbuzz(const int max) {
    if (max != 0) {
        fizzbuzz(max - 1);
    }

    if (max % 3 == 0 || max % 5 == 0) {
        if (max % 3 == 0) {
            std::cout << "Fizz";
        }
        if (max % 5 == 0) {
            std::cout << "Buzz";
        }
        std::cout << std::endl;
    } else {
        std::cout << max << std::endl;
    }
}

int main(const int argc, char* argv[]) {
    if (argc > 1) {
        fizzbuzz(std::stoi(argv[1]));
        return 0;
    }

    std::cerr << "Usage: " << argv[0] << " <number>" <<  std::endl;
    return 1;
}

File: coding/fizzbuzz/py/fizzbuzz.py
import sys

def fizzbuzz(n):
    if n != 0:
        fizzbuzz(n - 1)

    if n % 3 == 0:
        if n % 3 == 0 or n % 5 == 0:
            if n % 3 == 0:
                print("Fizz", end='')
            if n % 5 == 0:
                print("Buzz", end='')
    else:
        print(n, end='')
    print()

if len(sys.argv) > 1:
    fizzbuzz(sys.argv[1])
else:
    print("Usage: " + sys.argv[0] + " <number>", file=sys.stderr)
    exit(1)